### DISCUSSION TITLE
Blank window

### DISCUSSION THREAD

**Discussion by agmayuni** [flet-dev/flet#3864] [Q&A] [answered]
### Question

Just getting started. when i run the counter demo im having a blank window on Windows but works fine on a browser 

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Which version of Flet are you using?

**Reply by agmayuni** [reply]
version 0.23.2

**Reply by agmayuni** [reply, marked as answer]
found the solution. i downgraded to flet 0.21.2 and works fine on Windows 10
